test/system/src/test/java/io/pravega/test/system/StreamsAndScopesManagementTest.java

```diff
@@ -0,0 +1,254 @@
+
+@Slf4j
+@RunWith(SystemTestRunner.class)
+public class StreamsAndScopesManagementTest {
+
+    private static final int NUM_SCOPES = 5;
+    private static final int NUM_STREAMS = 20;
+    private static final int NUM_EVENTS = 100;
+    // TODO: Re-creation of Streams cannot be tested (Issue #2641).
+    private static final int TEST_ITERATIONS = 1;
+    @Rule
+    public Timeout globalTimeout = Timeout.seconds(8 * 60);
+
+    private final ScheduledExecutorService executor = ExecutorServiceHelpers.newScheduledThreadPool(4, "executor");
+
+    private URI controllerURI = null;
+    private StreamManager streamManager = null;
+    private Controller controller;
+
+    private Map<String, List<Long>> controllerPerfStats = new HashMap<>();
+
+    /**
+     * This is used to setup the services required by the system test framework.
+     *
+     * @throws MarathonException When error in setup.
+     */
+    @Environment
+    public static void initialize() throws MarathonException {
+
+        // 1. Check if zk is running, if not start it.
+        Service zkService = Utils.createZookeeperService();
+        if (!zkService.isRunning()) {
+            zkService.start(true);
+        }
+
+        List<URI> zkUris = zkService.getServiceDetails();
+        log.debug("Zookeeper service details: {}", zkUris);
+        // Get the zk ip details and pass it to bk, host, controller.
+        URI zkUri = zkUris.get(0);
+
+        // 2. Check if bk is running, otherwise start, get the zk ip.
+        Service bkService = Utils.createBookkeeperService(zkUri);
+        if (!bkService.isRunning()) {
+            bkService.start(true);
+        }
+
+        List<URI> bkUris = bkService.getServiceDetails();
+        log.debug("Bookkeeper service details: {}", bkUris);
+
+        // 3. Start controller.
+        Service conService = Utils.createPravegaControllerService(zkUri);
+        if (!conService.isRunning()) {
+            conService.start(true);
+        }
+
+        List<URI> conUris = conService.getServiceDetails();
+        log.debug("Pravega controller service details: {}", conUris);
+
+        // 4.Start segmentstore.
+        Service segService = Utils.createPravegaSegmentStoreService(zkUri, conUris.get(0));
+        if (!segService.isRunning()) {
+            segService.start(true);
+        }
+
+        List<URI> segUris = segService.getServiceDetails();
+        log.debug("Pravega segmentstore service details: {}", segUris);
+    }
+
+    @Before
+    public void setup() {
+        Service conService = Utils.createPravegaControllerService(null);
+        List<URI> ctlURIs = conService.getServiceDetails();
+        controllerURI = ctlURIs.get(0);
+        streamManager = StreamManager.create(controllerURI);
+        controller = new ControllerImpl(ControllerImplConfig.builder()
+                                                            .clientConfig(ClientConfig.builder().controllerURI(controllerURI).build())
```

Comment:
indentation seems incorrect. Please fix it.

Reply:
Done.